app/services/exports/lettings_log_export_service.rb

```diff
@@ -55,6 +55,14 @@ def retrieve_resources_from_range(range, year)
         )
         .pluck(:id)
 
+      # these must be separate as activerecord struggles to join to two different name change tables in the same query
+      ids.concat(
```

Comment:
could look into seeing if this can be done in a single query though at a quick glance the only options I could find call for writing raw SQL